File: Documentation/5.4/Samples/python/ClientApi/Session/Configuration/MaxRequests.py
from examples_base import ExampleBase


class MaxRequests(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_max_requests(self):
        with self.embedded_server.get_document_store("MaxRequests") as store:
            with store.open_session() as session:
                # region max_requests_1
                session._max_number_of_requests_per_session = 50
                # endregion

        # region max_requests_2
        store.conventions.max_number_of_requests_per_session = 100
        # endregion

File: Documentation/5.4/Samples/php/ClientApi/Session/Configuration/MaxRequests.php
<?php

use RavenDB\Documents\DocumentStore;

class MaxRequests
{
    public function example(): void
    {
        $store = new DocumentStore();
        try {
            $session = $store->openSession();
            try {
                # region max_requests_1
                $session->advanced()->setMaxNumberOfRequestsPerSession(50);
                # endregion
            } finally {
                $session->close();
            }

            # region max_requests_2
            $store->getConventions()->setMaxNumberOfRequestsPerSession(100);
            # endregion
        } finally {
            $store->close();
        }
    }
}
